src/lib/litegraph/src/LGraphCanvas.ts

```diff
@@ -3728,6 +3730,14 @@ export class LGraphCanvas
    */
   _pasteFromClipboard(
     options: IPasteFromClipboardOptions = {}
+  ): ClipboardPasteResult | undefined {
+    const data = localStorage.getItem('litegrapheditor_clipboard')
```

Comment:
https://vueuse.org/core/useLocalStorage/
You might be able to use the composable instead. Should help prevent typo based key misalignment.

Reply:
I think it'd also help with the [serialize/deserialize](https://vueuse.org/core/useStorage/#custom-serialization)